contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -45,6 +43,8 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
         mapping(uint256 snapshotId => mapping(address token => uint256 amount)) claimableAtSnapshot;
         mapping(uint256 snapshotId => mapping(address claimer => mapping(address token => bool))) isClaimedAtSnapshot;
         EnumerableSet.AddressSet tokens;
+        mapping(address token => uint256 accBalance) poolInfo;
```

Comment:
would rename `poolInfo` to `vaultInfo` given it's not a pool

Reply:
renamed to `vaultAccBalances`